### ISSUE TITLE
Expose scheduled charge/climate control as sensor

### ISSUE THREAD

**Issue by sammort** [Hyundai-Kia-Connect/hyundai_kia_connect_api#325]
Hey thanks for the great work on this API and integration with HA, it's really helpful to be able to setup some more advanced automations.

I hoped it might be possible to see the status of scheduled charging/climate control as a binary sensor in HA. My use case is this:

I usually only want my EV to charge during the 'off-peak' tariff, and this mostly works well when scheduled charging is set and enabled in the Bluelink app. However, frustratingly the setting will (seemingly at random) reset and scheduled charging is no longer enabled. This usually results in someone plugging in the car and not noticing that it immediately starts to charge (costing an arm and a leg with the current energy prices!)

It would be great to be able to detect this 'reset' using this API's integration with HA, by exposing this setting as a sensor in the app, would this be possible?

(This is similar to request in https://github.com/Hyundai-Kia-Connect/hyundai_kia_connect_api/issues/18)

**Comment by cdnninja** [comment]
What region are you in? I think most of this was added already but may not be all regions.

**Comment by sammort** [comment]
I'm in the EU, so I can see things like 1st Scheduled Departure Time, 2nd Scheduled Departure Time and Off Peak Start and End Time are alreayd implemented.

Some things that don't currently seem available, that are in the Bluelink app:
  1. Whether 'Scheduled for charging' and 'Scheduled climate control' is enabled/disabled
  2. Within 'Scheduled for charging' there are options to set off-peak timeframe (already exposed) and also whether 'Prioritisation' is 'Off-peak tariffs prioritised' or 'Off-peak tariffs only'

I find that when the car/API resets my setting to only charge during off-peak hours, the setting 'Scheduled for charging' will be disabled. If this was exposed as a binary sensor I could have some automation based on that sensor switching from true to false

(Let me know if it would be useful to have screenshots from the EU Bluelink app)

**Comment by sammort** [comment]
That's correct, in the Scheduled charging configuration is where you are able to set the 1st and 2nd departure schedules

**Comment by cdnninja** [comment]
Could you screenshot what it shows in app that is missing?   Today we have a binary sensor that should display for first and second departure schedules.  This is showing if enabled.
